core/storage/statistics/gae_models.py

```diff
@@ -2057,21 +2055,26 @@ def get_all_models(
                         shard_id)
                     for shard_id in range(
                         1, main_shard.shard_count + 1)]
-                all_models += cast(
-                    List[StateAnswersModel], cls.get_multi(shard_ids))
+                state_answer_models = cls.get_multi(shard_ids)
+                for state_answer_model in state_answer_models:
+                    # Filtering out the None cases for MyPy type checking.
```

Comment:
Needs more explanation (as usual) about how we know for certain it can't be None, and some discussion about why asserting rather than erroring is the right thing to do.

Reply:
Comment is updated!